docs/upgrade/v1-1-2-to-v1-2-0.md

```diff
@@ -199,6 +199,79 @@ A misconfigured PDB could cause this issue. To check if that's the case, perform
 - Related issue:
   - [[BUG] 3 Node AirGapped Cluster Upgrade Stuck v1.1.0->v1.1.2-rc4](https://github.com/harvester/harvester/issues/3730 )
 
+#### 5.3 The `instance-manager-e` pod could not be drained
+
+In some scenarios, during an upgrade, you might encounter an issue where the `instance-manager-e` pod cannot be drained. This situation is typically indicated by error messages in the Rancher logs, like the ones shown below:
+
+```
+$ kubectl logs deployment/rancher -n cattle-system | grep "evicting pod"
+evicting pod longhorn-system/instance-manager-r-a06a43f3437ab4f643eea7053b915a80
+evicting pod longhorn-system/instance-manager-e-452e87d2
+error when evicting pods/"instance-manager-r-a06a43f3437ab4f643eea7053b915a80" -n "Longhorn-system" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
+error when evicting pods/"instance-manager-e-452e87d2" -n "longhorn-system" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
+```
+
+Then, you need to check the `instance-manager-e` to see if any engine instances remain.
+
+```
+$ kubectl get instancemanager instance-manager-e-452e87d2 -n longhorn-system -o yaml | yq -e ".status.instances"
+pvc-7b120d60-1577-4716-be5a-62348271025a-e-1cd53c57
```

Comment:
The trailing colon of the engine name seems to be missing in the first line of the command output.

```suggestion
pvc-7b120d60-1577-4716-be5a-62348271025a-e-1cd53c57:
```